Rock.Encryption/CompositeCrypto.cs

```diff
@@ -129,7 +131,15 @@ public IDecryptor GetDecryptor(object keyIdentifier)
         /// </returns>
         public bool CanEncrypt(object keyIdentifier)
         {
-            return GetEncryptCrypto(keyIdentifier) != null;
+            try
+            {
+                GetEncryptCrypto(keyIdentifier);
+            }
+            catch (KeyNotFoundException)
+            {
```

Comment:
do we care to log this exception?  Any value in this being an Info/Warn?

Reply:
I don't think so, since this method is just asking whether you're able to encrypt. If you attempt to actually encrypt with that keyIdentifier, that's when you'd get an exception.